arm-servicefabric/2016-09-01/examples/ClusterPatchOperation_exmaple.json

```diff
@@ -0,0 +1,195 @@
+{
```

Comment:
Can you provide minimum input for the operation as well? Is all of this information required patch?